tests/pvc_utils.go

```diff
@@ -0,0 +1,74 @@
+package tests
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	defaultTimeout = 30*time.Second
+)
+
+// Creates a standard PVC in the passed in namespace.
+func CreateDefaultPVC(client *kubernetes.Clientset, namespace string, name string, size string) *k8sv1.PersistentVolumeClaim {
```

Comment:
It seems like we'd want to pass in the annotations here as well. I'm not sure what use this function is without PVC annotations.

Reply:
Its to create a plain jane PVC without annotations, which might be something we want to do.

Reply:
I guess I don't see what the value is in a plain-jane pvc for cdi testing. All cdi pvcs need annotations to define the endpoint.

Reply:
Negative test that creating a normal PVC doesn't trigger something in the controller?

Reply:
But for that case we can simply manually do it. So yeah I guess there is no real benefit

Reply:
I hadn't considered negative testing, so good point. We probably do want some e2e tests that ensure the controller only pays attention to cdi annotated pvcs.